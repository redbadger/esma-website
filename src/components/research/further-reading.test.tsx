import React from "react";
import renderer from "react-test-renderer";
import PrevNext from "./prev-next";
import FurtherReading from "./further-reading";

describe("FurtherReading", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <FurtherReading>
          <ul>
            <li>Sample entry</li>
          </ul>
        </FurtherReading>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
