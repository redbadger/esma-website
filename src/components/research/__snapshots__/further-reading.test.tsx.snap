// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FurtherReading renders correctly 1`] = `
<div
  className="further-reading-container css-1e497tz-furtherReadingStyles"
  role="list"
>
  <ul>
    <li>
      Sample entry
    </li>
  </ul>
</div>
`;
